Week-13/weather/middlewares/asyncHandler.js

```diff
@@ -0,0 +1,4 @@
+const asyncHandler = (fn) => (req, res, next) =>
```

Comment:
Will this handle multiple parallel requests?

Reply:
this is kind of try-catch wrapper.